src/prefect/cli/orion.py

```diff
@@ -150,6 +152,42 @@ async def start(
     app.console.print("Orion stopped!")
 
 
+@orion_app.command()
+async def status(
+    wait: bool = typer.Option(
+        False,
+        "--wait",
+        help="Runs health check loop until successful or optional timeout value is exceeded.",
+    ),
+    timeout: int = typer.Option(
+        None, "--timeout", help="Optional Timeout value in seconds"
+    ),
+):
+    """Verify Connection status with Orion API"""
+    async with get_client() as client:
+        status_check = True
+        start_time = pendulum.now()
+        while status_check == True:
+            response = await client.api_healthcheck()
+            if not response:
+                print("Orion Healthy")
+                status_check = False
+            else:
+                if wait == False:
+                    print(response)
```

Comment:
You'll want to use `app.console.print`

Reply:
See also the `exit_with_success` and `exit_with_failure` utils

Reply:
Updated to use exit_with_failure, the code looks much cleaner now, Thanks for the suggestions 😄